### PR TITLE
Update links in module API docs (generated from module.c)

### PR DESCRIPTION
**Pull request by zuiderkwast**
These are used in the docs and on the website.

### CODE REVIEW

**Review by zuiderkwast — COMMENTED** [review]
This is comments used for generated docs. I'm merging.

File: utils/generate-module-api-doc.rb [hunk 1]
```diff
@@ -160,7 +160,7 @@ def is_func_line(src, i)
 end
 
 # Populate the 'since' map (name => version) if we're in a git repo.
-require File.dirname(__FILE__) ++ '/module-api-since.rb'
+require "./" ++ File.dirname(__FILE__) ++ '/module-api-since.rb'
```
Comment (madolson) [inline utils/generate-module-api-doc.rb:163]: Why was this changed?

Comment (zuiderkwast) [reply utils/generate-module-api-doc.rb:163]: I don't know ruby that well, but with this change I can call the script regardless of my current directory.

Comment (zuiderkwast) [reply utils/generate-module-api-doc.rb:163]: `File.dirname(__FILE__)` is apparently a _relative_ path.

Comment (madolson) [reply utils/generate-module-api-doc.rb:163]: Ok, I suppose in fullness of time it should be called from automation.

**Review by madolson — APPROVED** [review]
